modules/tinymce/src/core/main/ts/api/util/I18n.ts

```diff
@@ -123,6 +123,7 @@ const translate = (text: Untranslated): TranslatedString => {
   };
 
   const removeContext = (str: string) => str.replace(/{context:\w+}$/, '');
+  const replaceWithEllipsisChar = (text: string) => text.replaceAll('...', '…');
```

Comment:
could it make sense to have this character in a constant like the whitespaces? 🤔